README.md

```diff
@@ -41,28 +41,28 @@ sktime is a library for time series analysis in Python. It provides a unified in
 [roadmap]: https://www.sktime.net/en/latest/roadmap.html
 [related software]: https://www.sktime.net/en/latest/related_software.html
 
-## :speech_balloon: Where to ask questions
+## :speech_balloon: Where to ask questions!
 
-Questions and feedback are extremely welcome! Please understand that we won't be able to provide individual support via email. We also believe that help is much more valuable if it's shared publicly, so that more people can benefit from it.
+We welcome questions and feedback and appreciate your understanding that we are unable to offer personalized assistance via email. We strongly believe in the value of sharing help publicly, as it allows a wider audience to benefit from it.
 
 | Type                            | Platforms                               |
 | ------------------------------- | --------------------------------------- |
 | :bug: **Bug Reports**              | [GitHub Issue Tracker]                  |
 | :sparkles: **Feature Requests & Ideas** | [GitHub Issue Tracker]                       |
 | :woman_technologist: **Usage Questions**          | [GitHub Discussions] · [Stack Overflow] |
 | :speech_balloon: **General Discussion**        | [GitHub Discussions] |
-| :factory: **Contribution & Development** | contributors channel · [Discord] |
-| :globe_with_meridians: **Community collaboration session** | [Discord] - Fridays 3pm UTC, dev/meet-ups channel |
+| :factory: **Contribution & Development** | Contributors channel · [Discord] |
```

Comment:
the channel name is actually `dev-chat`, how about saying "`dev-chat` channel"

Reply:
Ya, that sounds great!